src/shared/filesystemUtilities.ts

```diff
@@ -147,3 +148,19 @@ export function getNonexistentFilename(dir: string, name: string, suffix: string
         }
     }
 }
+
+/**
+ * Searches for existence of at least one file with the passed extension
+ * @param dir Directory to search
+ * @param extension string extension to look for (".ts")
+ * @param exclude GlobPattern - Pattern to ignore (ex. node_modules)
+ * @returns Boolean - true if at least one file is found with given extension
```

Comment:
Do not repeat the types in the docstring, this is already provided by the signature. And it risks drift if the signature changes later (then the docstring is just wrong).